src/Symfony/Component/Ldap/Adapter/ExtLdap/ConnectionOptions.php

```diff
@@ -23,6 +23,9 @@
  */
 final class ConnectionOptions
 {
+    const X_TLS_CRL_NONE = 0;
+    const X_TLS_CRL_PEER = 1;
+    const X_TLS_CRL_ALL = 2;
```

Comment:
Should use 0x... and be sorted with other X_* consts to me.

Reply:
done